gnovm/stdlibs/std/native.go

```diff
@@ -150,6 +152,22 @@ func X_encodeBech32(prefix string, bytes [20]byte) string {
 	return b32
 }
 
+func X_verifySignature(pubKeySigner string, msg string, signature string) (bool, string) {
```

Comment:
Please, let's rename this method to something more descriptive, since this is the sig verification method for both ed25519 and secp256k1. What do you think about `VerifyECSignature`, as the name for the native method?

Also, I'd rename the args here:
- `pubKeySigner` -> `bech32PubKey` (to be explicit)
- `msg` -> `rawData` (and please make this a `[]byte`)
- change the type of `signature` to `[]byte`